E2E/specs/example.spec.js

```diff
@@ -0,0 +1,18 @@
+const env = "local";
+let url = "";
+if (env === "local") {
+  url = "http://127.0.0.1:9000/#/";
```

Comment:
I think that this url should be in different file, because we need this conf for all tests